### ISSUE TITLE
[bug] AppRegistryNotReady in dev after deploy

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/openwisp-monitoring#174]
@nepython I tried deploying the `dev` branch on a VM and got this when trying to run migrations (or any other management command)
```
Error in sys.excepthook:
Traceback (most recent call last):
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 193, in _add_installed_apps_translations
    app_configs = reversed(list(apps.get_app_configs()))
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/apps/registry.py", line 144, in get_app_configs
    self.check_apps_ready()
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/apps/registry.py", line 135, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/openwisp2/env/lib/python3.8/site-packages/raven/utils/serializer/manager.py", line 76, in transform
    return repr(value)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/functional.py", line 91, in __repr__
    return repr(self.__cast())
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/functional.py", line 136, in __cast
    return self.__text_cast()
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/functional.py", line 124, in __text_cast
    return func(*self.__args, **self.__kw)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/__init__.py", line 92, in gettext
    return _trans.gettext(message)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 354, in gettext
    _default = _default or translation(settings.LANGUAGE_CODE)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 267, in translation
    _translations[language] = DjangoTranslation(language)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 154, in __init__
    self._add_installed_apps_translations()
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 195, in _add_installed_apps_translations
    raise AppRegistryNotReady(
django.core.exceptions.AppRegistryNotReady: The translation infrastructure cannot be initialized before the apps registry is ready. Check that you don't make non-lazy gettext calls at import time.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/openwisp2/env/lib/python3.8/site-packages/raven/utils/serializer/manager.py", line 94, in transform
    return serializer.transform(value, **kwargs)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/raven/utils/serializer/manager.py", line 78, in transform
    logger.exception(e)
  File "/usr/lib/python3.8/logging/__init__.py", line 1467, in exception
    Convenience method for logging an ERROR with exception information.
  File "/usr/lib/python3.8/logging/__init__.py", line 1469, in exception
    self.error(msg, *args, exc_info=exc_info, **kwargs)
  File "/usr/lib/python3.8/logging/__init__.py", line 1455, in error
    Log 'msg % args' with severity 'ERROR'.
  File "/usr/lib/python3.8/logging/__init__.py", line 1463, in error
    self._log(ERROR, msg, args, **kwargs)
  File "/usr/lib/python3.8/logging/__init__.py", line 1577, in _log
    self.handle(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 1587, in handle
    self.callHandlers(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 1649, in callHandlers
    hdlr.handle(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 950, in handle
    self.emit(record)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/log.py", line 120, in emit
    message = "%s\n\n%s" % (self.format(no_exc_record), reporter.get_traceback_text())
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/views/debug.py", line 341, in get_traceback_text
    return t.render(c)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/template/base.py", line 936, in render
    bit = node.render_annotated(context)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/template/base.py", line 903, in render_annotated
    return self.render(context)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/template/base.py", line 986, in render
    output = self.filter_expression.resolve(context)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/template/base.py", line 697, in resolve
    new_obj = func(obj, *arg_vals)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/template/defaultfilters.py", line 701, in date
    return formats.date_format(value, arg)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/formats.py", line 152, in date_format
    return dateformat.format(value, get_format(format or 'DATE_FORMAT', use_l10n=use_l10n))
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/formats.py", line 122, in get_format
    for module in get_format_modules(lang):
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/formats.py", line 92, in get_format_modules
    _format_modules_cache[lang] = list(iter_format_modules(lang, settings.FORMAT_MODULE_PATH))
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/formats.py", line 62, in iter_format_modules
    if not check_for_language(lang):
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/__init__.py", line 260, in check_for_language
    return _trans.check_for_language(lang_code)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 450, in check_for_language
    for path in all_locale_paths()
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 427, in all_locale_paths
    for app_config in apps.get_app_configs():
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/apps/registry.py", line 144, in get_app_configs
    self.check_apps_ready()
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/apps/registry.py", line 135, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.

Original exception was:
Traceback (most recent call last):
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 193, in _add_installed_apps_translations
    app_configs = reversed(list(apps.get_app_configs()))
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/apps/registry.py", line 144, in get_app_configs
    self.check_apps_ready()
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/apps/registry.py", line 135, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/core/management/__init__.py", line 377, in execute
    django.setup()
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/apps/config.py", line 90, in create
    module = import_module(entry)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/openwisp2/env/lib/python3.8/site-packages/openwisp_monitoring/monitoring/__init__.py", line 1, in <module>
    from .charts import register_chart, unregister_chart  # noqa
  File "/opt/openwisp2/env/lib/python3.8/site-packages/openwisp_monitoring/monitoring/charts.py", line 77, in <module>
    'unit': f' {_("ms")}',
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/functional.py", line 124, in __text_cast
    return func(*self.__args, **self.__kw)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/__init__.py", line 92, in gettext
    return _trans.gettext(message)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 354, in gettext
    _default = _default or translation(settings.LANGUAGE_CODE)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 267, in translation
    _translations[language] = DjangoTranslation(language)
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 154, in __init__
    self._add_installed_apps_translations()
  File "/opt/openwisp2/env/lib/python3.8/site-packages/django/utils/translation/trans_real.py", line 195, in _add_installed_apps_translations
    raise AppRegistryNotReady(
django.core.exceptions.AppRegistryNotReady: The translation infrastructure cannot be initialized before the apps registry is ready. Check that you don't make non-lazy gettext calls at import time.
```

When removing [this line](https://github.com/openwisp/openwisp-monitoring/blob/dev/openwisp_monitoring/monitoring/__init__.py#L1) the application can start.

I have no idea why this bug is not happening during development.
Why is that file being imported?
If there's anything that needs to be registered, do it in the ready() method of the AppConfig.

**Comment by nepython** [comment]
>I have no idea why this bug is not happening during development.

Yes, I can't recreate it in development but the bug makes sense.
>When removing this line the application can start.

Ok, this line is not doing anything but easing the import. I had added it after the review here https://github.com/openwisp/openwisp-monitoring/pull/117#discussion_r436075290. So, removing it would mean the below simple change
```python
from openwisp_monitoring.monitoring import register_chart_configuration   # currently
```
**to**
```python
from openwisp_monitoring.monitoring.charts import register_chart_configuration
```

**Comment by nemesisdesign** [comment]
Closed by 5094797
